src/openepd/api/errors.py

```diff
@@ -55,9 +55,20 @@ def __init__(
         super().__init__(http_status, error_summary, response, error_code)
         self.validation_errors: dict[str, list[str]] = validation_errors or {}
 
+    @staticmethod
+    def __flatten(nested_errors: dict[str, dict[str, list[str]] | list[str]]) -> dict[str, list[str]]:
+        result: dict[str, list[str]] = {}
+        for code, errors in nested_errors.items():
```

Comment:
Just to double check - this does work with multi-level errors like we saw (publishers > 0 > Owner > name) for example?